juriscraper/opinions/united_states/territories/pr.py

```diff
@@ -0,0 +1,75 @@
+"""
+Author: William Palin
+Date created: 2023-01-16
+Scraper for the Decisiones del Tribunal Supremo
+CourtID: ariz
```

Comment:
This ain't right.

Reply:
I'll sheepishly say... cause it's a draft!

Reply:
Oops, I thought I hadn't sent this when I realized it was a draft. My bad. Carry on!